plugins/modules/user.py

```diff
@@ -237,12 +250,24 @@ def change_password(module):
     '''
     name = module.params['name']
     passwd = module.params['password']
+    cleartext_passwd = module.params['cleartext_passwd']
     change_passwd_on_login = module.params['change_passwd_on_login']
-
-    if change_passwd_on_login:
-        cmd = "echo \'{user}:{password}\' | chpasswd -ec".format(user=name, password=passwd)
-    else:
+    
+    if passwd and cleartext_passwd:
+        msg = "\Specify either 'passwd' and 'cleartext_passwd', not both"
+        module.fail_json(msg=msg, rc=pass_rc, stdout=pass_out, stderr=pass_err)
+        
+    if passwd:    
+      if change_passwd_on_login:
         cmd = "echo \'{user}:{password}\' | chpasswd -e".format(user=name, password=passwd)
+      else:
+        cmd = "echo \'{user}:{password}\' | chpasswd -ec".format(user=name, password=passwd)
+        
+    if cleartext_passwd:    
+      if change_passwd_on_login:
+        cmd = "echo \'{user}:{cleartext_passwd}\' | chpasswd".format(user=name, password=cleartext_passwd)
```

Comment:
"cleartext_passwd=cleartext_passwd" should be used in the format statement and not "passwd=cleartext_passwd".